settings/security/third-party/saml.rst

```diff
@@ -141,16 +152,39 @@ IDP Certificate fingerprint
       **Keycloak users:** Find your certificate in the Keycloak admin panel
       under **Realm Settings > Keys > Algorithm: RS256 > Certificate**.
 
-Name Identifier format
+Name identifier format
    This is the unique identifiers field type. Usually should be
    ``urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress`` in any case.
 
    .. warning::
 
       Zammad expects an email address as unique identifier!
 
+UID attribute name
+   Here you can define an attribute that uniquely identifies the user. If unset,
+   the name identifier returned by the IDP is used.
+
+SSL verification
+   Here you can decide if the certificate of the IdP server has to be
+   verified or not (default: ``yes``).
```

Comment:
This needs to be rephrased. It's all about the connection to the provider service.